### PR TITLE
[BKR-66] Avoid Round Down on Withdrawal Fees

### PR DESCRIPTION
**Pull request by hvasconcelos**

### CODE REVIEW

File: scripts/common.ts [hunk 1]
```diff
@@ -57,9 +57,14 @@ export async function deployVault(
   tokenSymbol: string,
   serviceRegistry: string,
   strategy: string,
-  proxyAdmin?: any
+  mathLibrary: any,
+  proxyAdmin?: any,
```
Comment (ickas) [inline scripts/common.ts:61]: Strings?

File: scripts/deploy.ts [hunk 2]
```diff
@@ -190,7 +197,7 @@ async function changeSettings(
   spinner: any,
   settingsAddress: string,
   strategyAddress: string,
-  vaultAddress: string
+  vaultAddress: string,
 ) {
   const settings = await ethers.getContractAt("Settings", settingsAddress);
```
Comment (ickas) [inline scripts/deploy.ts:202]:
```suggestion
```

File: scripts/tasks.ts
Comment (ickas) [inline scripts/tasks.ts:1]: You can delete the unused `ErrorDecoder` on line `2`
```
import { ErrorDecoder } from 'ethers-decode-error'
```

Comment (ickas) [inline scripts/tasks.ts:1]: You can delete the unused `signers` on line `90`

```
const signers = await ethers.getSigners();
```

File: test/core/VaultWstETH.ts
Comment (ickas) [inline test/core/VaultWstETH.ts:1]: Unused code on lines:

`175` anotherAccount
`236` otherAccount
`238` anotherAccount
`288` settings
`519` chainId
`642` anotherAccount

File: test/core/common.ts
Comment (ickas) [inline test/core/common.ts:1]: Unused code on line `71`
```
const oracle = await deployCbETHToUSDOracle(serviceRegistry, config.pyth);
```

File: scripts/common.ts (hunk 1 above)
Comment (hvasconcelos) [reply scripts/common.ts:61]: no

**Review by ickas — APPROVED** [review]

**Pull request merged**